source/Calamari.Tests/KubernetesFixtures/KubernetesRunsScriptFixture.cs

```diff
@@ -238,7 +238,6 @@ public void ExecutionWithEKS_IAMAuthenticator()
         }
         
         [Test]
-        [WindowsTest] // This test requires the aws cli tools. Currently only configured to install on Windows
```

Comment:
This is not true anymore, I don't think. I've removed it to see if the test passes on linux.

Reply:
I was wrong - linux needs sudo to install the tool, which blows up the test on the workers. 😞

Reply:
This is interesting because we are using `sudo` in https://github.com/OctopusDeploy/Calamari/blob/d203d663372dd02ae33920ab420eabf2e3a5e99a/source/Calamari.Tests/KubernetesFixtures/InstallTools.cs#L166
Couldn't we do the same here too @scottmerchant ?

Reply:
Ah yep I guess so - I didn't look to much into it as originally it wasn't running on linux so I just thought I'd try and see how it goes. I think it's probably worth just getting this change in to get the build green so Yihao can get his PR going. I'll fix the cli version as you suggested though 👍